references/cli/delete.go

```diff
@@ -76,15 +76,21 @@ func NewDeleteCommand(c common2.Args, order string, ioStreams cmdutil.IOStreams)
 			return err
 		}
 		o.ForceDelete = force
-
+		userInput := NewUserInput()
 		if svcname == "" {
-			ioStreams.Infof(fmt.Sprintf("Deleting Application \"%s\"\n", o.AppName))
```

Comment:
Why we need the user confirmation here? For double check?

Reply:
The delete action is dangerous, I always made mistaken deletions, especially using the wrong namespace. `There is a hidden context namespace setting in Vela`
Here remind users to double-check the application name and namespace.